lightkurve/lightcurve.py

```diff
@@ -306,7 +306,10 @@ def show_properties(self):
         attrs = {}
         for attr in dir(self):
             if not attr.startswith('_'):
-                res = getattr(self, attr)
+                try:
+                    res = getattr(self, attr)
+                except Exception:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Try, Except, Continue detected.](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4743301)

Reply:
This is the best I could come up with. Seems robust enough for a function that is only meant to print some useful information.  Otherwise incompatible with how the `astropy_time` property is defined to raise `ValueError`.

Reply:
Agreed, this is fine!